src/Smalot/PdfParser/Parser.php

```diff
@@ -58,28 +58,18 @@ class Parser
 
     protected $rawDataParser;
 
-    public function __construct($cfg = [], Config $config = null)
+    public function __construct($cfg = [], ?Config $config = null)
     {
         $this->config = $config ?: new Config();
         $this->rawDataParser = new RawDataParser($cfg, $this->config);
     }
 
-    /**
-     * @return Config
-     */
-    public function getConfig()
+    public function getConfig(): Config
     {
         return $this->config;
     }
 
-    /**
-     * @param string $filename
-     *
-     * @return Document
-     *
-     * @throws \Exception
```

Comment:
Please keep this line.

Reply:
Same as before, only keep if function itself can throw an exception.

Reply:
I changed this. The called `parseContent` method threw an exception and there was a big comment there (I missed it before), which said that users of the library want the `@throws \Exception` PHPDoc on this particular function.